.github/workflows/ci.yaml

```diff
@@ -17,17 +17,16 @@ jobs:
 
       - uses: supabase/setup-cli@v1
         with:
-          version: 1.0.0
+          version: latest
 
-      - run: supabase start
-
-      # - run: supabase db lint
-      # - run: supabase db test
+      - run: supabase db start
+      - run: supabase db lint
+      - run: supabase test db
 
       - name: Verify generated types are checked in
         run: |
           supabase gen types typescript --local > types.gen.ts
```

Comment:
Thank you for providing an amazing demo.👍 

However, I'm curious about the reason for using the `--local` flag to generate `types.gen.ts.` It's not explained in the documentation at [https://supabase.com/docs/guides/api/rest/generating-types](https://supabase.com/docs/guides/api/rest/generating-types), so I'm wondering.

Reply:
Sure, the `gen types` command can target different databases to read its schema. The `--local` flag targets the local database, which is created from supabase db start.

Other targets include `--linked` or `--db-url` which lets you generate types from a remote project. You can find more explanation in our reference doc https://supabase.com/docs/reference/cli/supabase-gen-types-typescript#supabase-gen-types-typescript